scripts/node-addon/update-and-push.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
@calebmshafer suggest a better location for these scripts?

Reply:
I'd put it in `common/scripts` unless you think it's too hidden there.

Reply:
moved.